styleguide/components/31-molecules/search/_search.scss

```diff
@@ -0,0 +1,11 @@
+.search {
+  display: flex;
+  position: relative;
+  flex-wrap: wrap;
+
+  label {
+    @extend %h5;
```

Comment:
This label does not provide sufficient information:
'are you looking for another page?' ... I would expect a checkbox.

Better not corrupt the label to look like a h5 and just use a heading element and use aria-label on the search field.
(we'll have to do this in the header too anyway).

Reply:
I changed it to a title and fixed aria-label of the input-search atom.